### ISSUE TITLE
Currently In Progress

### ISSUE THREAD

**Issue by Theawesomeboophis** [Theawesomeboophis/UndeadPeopleTileset#7]
Hello, this is a list of things currently being worked on that will be updated with latest news.

Currently In-Progress:

The One Repo To Rule Them All: combining unpacked and normal tileset repo into one repo for maintenance convenience.

Exodii Quadruped Resprite

Raptor Resprite

Removal and Respriting of Copyrighted Sprites

Visible Sky ( Suggested on Reddit)

Please keep in mind that these are in no particular order and have no set due date.

**Comment by talker2** [comment]
so,just to be sure your going to be drawing new sprites.

**Comment by Theawesomeboophis** [comment]
Yes, I plan on updating the tileset and adding new sprites.

**Comment by SirClarke** [comment]
Sweet Dude best wishes and I for one am looking forward to it

**Comment by Theawesomeboophis** [comment]
Unpacked is going to be updated for combining with this repo, I have already added xotto support to some unpacked mods to get rid of compatibility errors.

**Comment by toweroglass** [comment]
does that mean if you've got this installed and are running msxotto it'll use the UDP graphical mods?

**Comment by Theawesomeboophis** [comment]
> does that mean if you've got this installed and are running msxotto it'll use the UDP graphical mods?

Yes, all the mod tilesets in udp can now be used with xotto, meaning mod support will not be lost between the two. This applies to all default and kenan mods as well.

**Comment by Guderiss** [comment]
Sorry to bother you, but I would like to ask if there is a discord channel here. I would like to participate in the tileset drawing work, and I really like this tileset. This is the latest version of "UndeadPeopleTileset" that I have come into contact with, and it seems that it is still being updated.

**Comment by SirClarke** [comment]
https://github.com/Theawesomeboophis/UndeadPeopleTileset/issues/16 

I think this is the link

**Comment by Theawesomeboophis** [comment]
Issue 16 is indeed the link to the discord

**Comment by Theawesomeboophis** [comment]
We're currently working on flag sprites.
